<launch>

 <node pkg="metal_detector" type="metal_detector_node.py" name="metal_detector" output="screen">
      <remap from="/metal_detector/measurements" to="/vrep/metalDetector"/>
      <param name="world_frame" value="/world" />
      <param name="sensor_frame" value="/VSV/Tool"/>
      <param name="Np" value="1000" />
      <param name="min_displacement" value="0.01" />
      <param name="min_measurement" value="0.15" />
  </node>


</launch>
